src/compiler/scala/tools/nsc/typechecker/Infer.scala

```diff
@@ -1276,8 +1276,8 @@ trait Infer extends Checkable {
      *  If no alternative matches `pt`, take the parameterless one anyway.
      */
     def inferExprAlternative(tree: Tree, pt: Type): Tree = {
-      val c = context
-      class InferTwice(pre: Type, alts: List[Symbol]) extends c.TryTwice {
+      val `context'` = context
+      class InferTwice(pre: Type, alts: List[Symbol]) extends `context'`.TryTwice {
```

Comment:
Yes, I got carried away. But it's also annoying to fire up an IDE just to see where the letter `c` is used. Since it's just for the stable id alias, it's a use case for `prime'`. So, nothing progresses without taking some risks.